<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="abriArchivo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>file.png</normaloff>file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="inicio">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cambiarAGestual">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>0</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modo gestual</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreArchivo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Leer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPausar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pausar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReproducir">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reproducir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTerminar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMicro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>abierto.png</normaloff>abierto.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
